numpy_questions.py

```diff
@@ -62,6 +66,10 @@ def wallis_product(n_terms):
     pi : float
         The approximation of order `n_terms` of pi using the Wallis product.
     """
-    # XXX : The n_terms is an int that corresponds to the number of
-    # terms in the product. For example 10000.
-    return 0.
+    pi = 1
+    if n_terms == 0:
```

Comment:
cosmetic comment: range(1, 1) is empty so you could omit this case and the code would do the exact same thing